README.md

```diff
@@ -793,24 +793,30 @@ machine = Machine(model, states, transitions)
 #### <a name="diagrams"></a> Diagrams
 
 Additional Keywords:
-* `title` (optional): Sets the title of the generated image.
+* `title` (optional): Sets the title of the generated image
 * `show_conditions` (default False): Shows conditions at transition edges
+* `show_auto_transitions` (default False): Shows auto transitions in graph
 
-Transitions can generate basic state diagrams displaying all valid transitions between states. To use the graphing functionality, you'll need to have `pygraphviz` installed (`pip install pygraphviz`). With
- `GraphMachine` enabled, a PyGraphviz `AGraph` object is generated during machine initialization and is constantly updated when the machine state changes:
+Transitions can generate basic state diagrams displaying all valid transitions between states. To use the graphing functionality, you'll need to have `pygraphviz` installed (`pip install pygraphviz`). With `GraphMachine` enabled, a PyGraphviz `AGraph` object is generated during machine initialization and is constantly updated when the machine state changes:
 
 ```python
 from transitions.extensions import GraphMachine as Machine
 m = Model()
 machine = Machine(model=m, ...)
-m.graph.draw('my_state_diagram.png', prog='dot')
+# Machine(model=m, show_auto_transitions=True, ...)
```

Comment:
Did you mean to leave this line in here to show the available options? Would be clearer with a comment.